packages/core/strapi/lib/commands/transfer/import.js

```diff
@@ -1,13 +1,17 @@
 'use strict';
 
 const {
-  createLocalFileSourceProvider,
-  createLocalStrapiDestinationProvider,
+  providers: { createLocalFileSourceProvider },
+} = require('@strapi/data-transfer/lib/file');
+const {
+  providers: { createLocalStrapiDestinationProvider, DEFAULT_CONFLICT_STRATEGY },
+} = require('@strapi/data-transfer/lib/strapi');
+const {
   createTransferEngine,
   DEFAULT_VERSION_STRATEGY,
   DEFAULT_SCHEMA_STRATEGY,
-  DEFAULT_CONFLICT_STRATEGY,
-} = require('@strapi/data-transfer');
+} = require('@strapi/data-transfer/lib/engine');
+
```

Comment:
What's the reason for this instead of using @strapi/data-transfer? (same question for everywhere it's done)

Reply:
I was trying to only import what I needed. Initially, my goal was to really decouple everything and do imports like
```js
import {} from '@strapi/data-transfer/strapi';
```
But eslint wasn't happy even though it worked.

So then it was either 
```js
import { providers: { createLocalFileSourceProvider } } from '@strapi/data-transfer/strapi';

// or

import {  strapi: { providers: { createLocalFileSourceProvider } } } from '@strapi/data-transfer';
```

I don't have a strong opinion there

Reply:
I think the goal is to be able to have isolated imports, but I am not sure 100%

Reply:
As long as they still work at @strapi/data-transfer for those who prefer (and I checked, they do) I'm happy :)